src/video.cpp

```diff
@@ -1281,6 +1281,17 @@ void encode_run(
   const encoder_t &encoder,
   void *channel_data) {
 
+  // get minimum fps from config
+  auto MIN_FPS = std::chrono::milliseconds(*&config::video.min_fps * 10);
```

Comment:
The formula for converting framerate to milliseconds is the following

1000 / {{FRAMERATE}}

Additionally, it needs to do error checking to confirm min_fps is an integer and not something else.

Also, probably not necessary to do *& here but I don't have my IDE open here so sorry if I am wrong here..

Also we should rename this to minimum_frametime

Uppercase variables are usually convention of global constants

Reply:
I'll update the formula.

I'd guess Sunshine will error out far before we get to this point if the value is not an integer, based on the config structure, but I'll throw a check in anyway.